assets/javascripts/kitten/components/lists/list/components/button-item.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import COLORS from '../../../../constants/colors-config'
+import { pxToRem } from '../../../../helpers/utils/typography'
+import { ArrowIcon as KittenArrowIcon } from '../../../icons/arrow-icon'
+
+const Item = styled.div`
+  display: flex;
+  border: ${pxToRem(2)} solid ${COLORS.line1};
+  border-top: none;
+  cursor: pointer;
+  padding-left: ${pxToRem(20)};
+  padding-right: ${pxToRem(20)};
+
+  :hover {
+    background-color: ${COLORS.background2};
+  }
+
+  ${({ withTopBorder }) =>
+    withTopBorder &&
+    css`
+      border-top: ${pxToRem(2)} solid ${COLORS.line1};
+    `}
+
+  ${({ disabled }) =>
+    disabled &&
+    css`
+      color: ${COLORS.line2};
+      background-color: ${COLORS.background1};
+      cursor: not-allowed;
+
+      :hover {
+        background-color: ${COLORS.background1};
+      }
+    `}
+
+  ${({ styles }) => styles}
+`
+
+const Content = styled.div`
+  flex-grow: 1;
+`
+
+const Arrow = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const ArrowIcon = styled(({ focus, ...props }) => (
+  <KittenArrowIcon {...props} />
+))`
+  position: relative;
+  left: 0;
+  transition: left 0.2s;
```

Comment:
Qu'est ce que tu penses d'ajouter une `transition-timing-function` sur cette transition?

Reply:
C'est pas `ease` par défaut ? Tu veux en utiliser une autre ?

Reply:
My bad, `ease` est bien la valeur par défaut si rien n'est spécifié:
https://css-tricks.com/ease-out-in-ease-in-out/